lib/01_temperature.rb

```diff
@@ -0,0 +1,7 @@
+def ftoc(fahrenheit)
+  celsius = ((fahrenheit - 32) * 5.0) / 9
```

Comment:
You technically don't need the `celsius =` part because you aren't using that variable anywhere, so it's a bit pointless to define it. You could remove it and the method would still work the same, it would return the same number. Same with the the `ctof`, `add`, and `substract` methods below.